packages/sessions-sdk-rs/Cargo.toml

```diff
@@ -15,7 +15,7 @@ anchor = ["anchor-lang"]
 anchor-idl = ["anchor-lang/idl-build"]
 
 [dependencies]
-borsh = { version = ">=0.10.0, <2.0.0"}
+borsh = { version = ">=0.10.0, <1.0.0"}
```

Comment:
reconsider this, new versions of solana program want borsh 1.x.x

Reply:
Ah so the reason I set this down was that when I installed some packages, something updated borsh to 1.x.x, but the install failed due to some breaking changes in borsh.   I'd need to revisit to recall exactly what the issue was but I believe that something in the sessions sdk is actually incompatible with borsh 1.x.x